Stack/Opc.Ua.Core/Stack/Server/ServerBase.cs

```diff
@@ -1152,11 +1157,6 @@ private static string GetBestDiscoveryUrl(Uri clientUrl, BaseAddress baseAddress
                         continue;
                     }
 
-                    if (endpointUrl.Port != baseAddress.Url.Port)
```

Comment:
the method which is corrected here has a seemingly unused parameter clientUrl - do we want to change (remove) that?

Reply:
good catch, but I'd rather not change it since it is called from the server lib